python-lib/publish/action.yml

```diff
@@ -34,6 +34,12 @@ runs:
         feature-name: ${{ steps.feature.outputs.feature-name }}
       env:
         FULL_ACCESS_TOKEN: ${{ env.PYPI_FULL_ACCESS_TOKEN }}
+    - uses: LedgerHQ/actions/nexus/latest-version@main
+      if: inputs.public != 'true' && github.event_name == 'push' && startsWith(github.ref, 'refs/heads/feature/')
+      id: nexus
+      with:
+        package-name: ${{ steps.feature.outputs.package-name }}
+        feature-name: ${{ steps.feature.outputs.feature-name }}
```

Comment:
If we merge this, all the repositories using this action will start trying to publish on Nexus, but they are not passing the right env vars for this to succeed. So before merging you'd have to update all the repos first.

I think it will be easier for you if we add a `if: env.GREEN_NEXUS_HOST != ''` or something like that to make the action backward compatible.
Maybe even do the same for the Gemfury parts so that they slowly stop being used.